### ISSUE TITLE
Master server not disconnecting properly

### ISSUE THREAD

**Issue by baflink** [BeardedManStudios/ForgeNetworkingRemastered#70]
### Version Number and Operating System(s):
Latest Master branch

### Expected behavior:
When you play in unity then stop playing the server should be removed from the master server

### Actual behavior:
When you play and then stop playing then the server is not removed from the master server

### Steps to reproduce:
Setup master server
Turn on logging in the master server
Play in editor as server to register itself on master server
Stop playing in editor and observe that the server does not get removed

**Comment by pointcache** [comment]
That is not what i said tho, i said that while i have a server in build running.. ok
1. I build, run as host
2. can see server in master server
3. now press play in editor
4. press stop
5. server now dissapears from master, as if i closed the host, but the host is still running!

**Comment by baflink** [comment]
So you were running 2 servers on the same machine then @pointcache?

**Comment by pointcache** [comment]
No, the build is runing the server, because i hit Host on it, i didnt hit host in editor, i literally did nothing, just play/stop

**Comment by baflink** [comment]
Is this reproducible with 2 builds, one where you host and the other where you start it and then close it?

**Comment by pointcache** [comment]
I mistook the start/play as related to disconnect, testing shows that disconnect is just time related it seems.
Like it just falls off after some time

**Comment by baflink** [comment]
That was an issue that we had in the previous code, however this PR should have fixed that:  https://github.com/BeardedManStudios/ForgeNetworkingRemastered/commit/a9eea341cb3a97fe2f694b2bf41112efc83477bb

So I'll look into it again.

**Comment by Memorix101** [comment]
The master server doesn't work for me at all. No server is being registered.
